flux-bootstrap.sh

```diff
@@ -0,0 +1,8 @@
+export REPOUSER="marcpires"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/marcpiresrj/gitops/pullRequest?prid=5106344)